R/pkg/R/DataFrame.R

```diff
@@ -303,8 +303,28 @@ setMethod("colnames",
 #' @rdname columns
 #' @name colnames<-
 setMethod("colnames<-",
-          signature(x = "DataFrame", value = "character"),
```

Comment:
I agree - letting R do type matching for the method signature seems like a better approach?